docs/source/changelog.md

```diff
@@ -5,7 +5,32 @@ its link will bring up a GitHub listing of changes. Use `git log` on the
 command line for details.
 
 
-## [Unreleased]
+## [Unreleased] 0.7
+
+### Added
+- Implement Services API [\#705](https://github.com/jupyterhub/jupyterhub/pull/705)
```

Comment:
Minor style point - some Markdown processors (I think the one we use in the live notebook?) get confused if there isn't an empty line after headings. I like to be better safe than sorry, and always leave a blank line after headings.